src/aat/resources/features/F-1003/F-1003-V3/Default_Document_Upload_Data_01.td.json

```diff
@@ -0,0 +1,29 @@
+{
+  "_guid_": "Default_Document_Upload_Data_01",
+  "_extends_": "Default_Document_Upload_Data_Base",
+  "specs": [
+    "to upload a document with mandatory metadata"
+  ],
+  "request": {
+    "body": {
+      "arrayInMap": [
+        {
+          "key": "classification",
+          "value": "PUBLIC"
+        },
+        {
+          "key": "caseTypeId",
+          "value": "BEFTA_CASETYPE_2_2"
+        },
+        {
+          "key": "jurisdictionId",
+          "value": "BEFTA_JURISDICTION_2"
+        },
+        {
+          "key": "files",
+          "filePath": "features/common/files/Sample.doc"
+        }
+      ]
+    }
```

Comment:
in fact, there could be a nice way of eliminating all these repeated blocks, and have just the below lines:
```
  "_guid_": "Default_Document_Upload_Data_01",
  "_extends_": "Default_Document_Upload_Data_Base",
```

This is possible, and I can show you how, if you like.
Hint: The file names could be dynamically picked from a custom value with respect to the _guid_ value.